invenio_app_rdm/theme/assets/js/invenio_app_rdm/search/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,87 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2019 CERN.
+// Copyright (C) 2019 Northwestern University.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Input } from "semantic-ui-react";
+import {
+  UrlHandlerApi,
+  onQueryChanged as triggeronQueryChanged
+} from "react-searchkit/esm";
+
+export class SearchBar extends Component {
```

Comment:
question: (these are all just questions for me to better understand the architecture: not meant as critiques) Seeing the need for us to define `onKeyPress` below seems like a lot... are we doing very specific customizations here?

Reply:
This is just to enable on Enter search :) I think is quite useful to trigger the search on enter no? :)

Reply:
My puzzlement here, was exactly over *us* having to define the "search on enter" rather than react-searchkit doing it for us. I think this was answered by our discussion online: some of this will be moved to invenio-search-ui and *it* will be the one defining it for us. So all good.